docs/modules/trino/pages/usage-guide/catalogs/generic.adoc

```diff
@@ -31,6 +31,10 @@ spec:
           valueFromSecret:
             name: my-postgresql-credentials-secret
             key: password
+#         if tables are capitalized / upper case for e.g. frost-server which can be read by trino,
```

Comment:
I would not use it in the example, rather have e.g. a troubleshoot header where you mention postgres / mysql and the table names etc.
Frost should not be in here, its about postgres/mysql.

Reply:
Done